src/libraries/System.Private.CoreLib/src/System/Reflection/MethodBase.cs

```diff
@@ -121,6 +121,22 @@ internal static void AppendParameters(ref ValueStringBuilder sbParamList, Type[]
             }
         }
 
+        // helper method to construct the string representation of the parameter list
```

Comment:
```suggestion
```
This is used for more than just constructing string representation of the parameter list. For example, https://github.com/dotnet/runtime/blob/main/src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/CustomAttributeBuilder.cs#L85